### ISSUE TITLE
Neo-cli node synced from bootstrap package not accessing storage (TestNet)

### ISSUE THREAD

**Issue by georgearchon** [neo-project/neo-cli#342]
The key value pair exists on the blockchain:
```
curl -v -d '{"jsonrpc": "2.0","method": "getstorage","params": ["0x5d28b342011fc4b8bc9ee0a1d98f9bcb6f4c0326","74756573646179"],"id": 15}' http://seed3.ngd.network:20332

{"jsonrpc":"2.0","id":15,"result":"626f6f"}
```
But when calling local node synced from bootstrap package:
```
curl -v -d '{"jsonrpc": "2.0","method": "getstorage","params": ["0x5d28b342011fc4b8bc9ee0a1d98f9bcb6f4c0326","74756573646179"],"id": 15}' http://127.0.0.1:20332

{"jsonrpc":"2.0","id":15,"result":null}
```
The UI of the neo-cli says my local node is synced:
```
block: 2526400/2526400/2526400  connected: 12  unconnected: 0                   
 ```
The node is synced using bootstrap package from https://sync.ngd.network/ using the docs https://docs.neo.org/en-us/network/syncblocks.html

**Comment by vncoelho** [comment]
This node in the printscreen was synced using the bootstrap package:

![image](https://user-images.githubusercontent.com/17395762/55922991-f18b9b80-5bd9-11e9-91b0-103cc9b659c2.png)

It looks like it is working normally, @georgearchon.
```js
{"jsonrpc":"2.0","id":15,"result":"626f6f"}
```

This should be another problem.

**Comment by georgearchon** [comment]
Being that you know the code base, where do you think a problem could have arisen?
Such insight would be helpful.

**Comment by vncoelho** [comment]
@georgearchon, hard question...ahueahuea

I need more details about your system details, as well as the procedure you took.
Syncing Testnet is in general quite fast, I think that in a 1 core computer we are taking around 5 hours only using all plugins (storage debug, assets tracking, nep5 history, etc...)

We recently fixed problems that were crashing/corrupting db. However, there is still a PR open in the official leveldb repository: https://github.com/google/leveldb/pull/ 339
I include an extra space to avoid linking this issue with their discussion, since I am not sure about the source of the error here.
However, I suggest that you take a look at the last comment @jsolman left there. Your case could be the same.

I suggest that you sync it again, I never got such problem and I am syncing very often.

**Comment by georgearchon** [comment]
This problem is not an isolated issue with my setup.
Querying node running neo-cli version 2.10.0
```
curl -v -d '{"jsonrpc": "2.0","method": "getstorage","params": ["0x5d28b342011fc4b8bc9ee0a1d98f9bcb6f4c0326","74756573646179"],"id": 15}' http://seed5.neo.org:20332

{"jsonrpc":"2.0","id":15,"result":"626f6f"}
```

Querying node running neo-cli 2.10.1

```
curl -v -d '{"jsonrpc": "2.0","method": "getstorage","params": ["0x5d28b342011fc4b8bc9ee0a1d98f9bcb6f4c0326","74756573646179"],"id": 15}' http://seed5.ngd.network:20332


{"jsonrpc":"2.0","id":15,"result":null}
```

Looks like a bug got shipped with your update.

**Comment by vncoelho** [comment]
@georgearchon, it has already been solved on 2.10.2
https://github.com/neo-project/neo-vm/pull/126

**Comment by georgearchon** [comment]
@vncoelho any idea when 2.10.2 with the fix of this storage issue will be available?
